### ISSUE TITLE
Big data in basic and translational cancer research

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#12]
## TL;DR

Review the current state of the art and future challenges for harnessing big data to advance cancer research and treatment.

### Paper Link

https://www.nature.com/articles/s41540-022-00218-9

### Author/Institution

Peng Jiang (National Cancer Institute), 

## Overview

The subject of big data in cancer is of immense scope, and it is impossible to cover everything in one review. We therefore focus on key big-data analyses that led to conceptual advances in our understanding of cancer biology and impacted disease diagnosis and treatment decisions. Further, we detail reviews in the pertaining sections to direct interested readers to relevant resources. We acknowledge that our limited selection of topics and examples may omit important work, for which we sincerely apologize.

In this Review, we begin by describing major data sources. Next, we review and discuss data analysis approaches designed to leverage big datasets for cancer discoveries. We then introduce ongoing efforts to harness big data in clinically oriented, translational studies, the primary focus of this Review. Finally, we discuss current challenges and future steps to push forward big data use in cancer.

## Cite

Jiang, P., Sinha, S., Aldape, K. et al. Big data in basic and translational cancer research. Nat Rev Cancer 22, 625–639 (2022). https://doi.org/10.1038/s41568-022-00502-0
